spark/src/main/scala/geotrellis/spark/io/AttributeCaching.scala

```diff
@@ -26,10 +26,10 @@ import scala.concurrent.duration._
 
 
 trait AttributeCaching { self: AttributeStore =>
-  private val expiration = ConfigFactory.load().getInt("geotrellis.attribute.caching.expirationMinutes")
-  private val maxSize = ConfigFactory.load().getInt("geotrellis.attribute.caching.maxSize")
+  private def expiration = ConfigFactory.load().getInt("geotrellis.attribute.caching.expirationMinutes")
+  private def maxSize = ConfigFactory.load().getInt("geotrellis.attribute.caching.maxSize")
 
-  private val cache =
+  private final val cache =
```

Comment:
Still a little uncomfortable adding these as private fields when we don't have to:

```scala
private final val cache = {
  val config = ConfigFactory.load()
  val expiration = config.getInt("geotrellis.attribute.caching.expirationMinutes")
...
}
```
would remove all possible concern.